src/main/java/ejercicio/banco/service/PaymentService.java

```diff
@@ -7,8 +7,8 @@
 import java.util.List;
 
 public interface PaymentService {
-	
-	public List<Payment> manage(String filename, Bank bestBank) throws FileNotFoundException;
-	
-	public void fwriter(String filename, Bank bestBank);
+
+    public List<Payment> manage(String filename, Bank bestBank) throws FileNotFoundException;
+
+    public void fwriter(String filename, Bank bestBank);
```

Comment:
This method name is not really descriptive. Ideally, we need to be as concise as possible when selecting method names.